app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,20 @@
+class AdminsController < ApplicationController
+  def index
+    @shelters = Shelter.reverse_alphabetical_name
+  end
+
+  def show
+    @application = Application.find(params[:id])
+    @pets = @application.pets
```

Comment:
I would remove this from your controller and just call 
```ruby
@application.pets.each do
```
in your view. We want to keep our controllers lean.